kythe/cxx/indexer/cxx/testdata/rec/rec_agg_init.cc

```diff
@@ -1,14 +1,34 @@
 // Tests that aggregate initialization references struct.
 
 //- @S defines/binding StructS
-struct S {};
+struct S {
+  //- @a defines/binding FieldA
+  int a;
+  //- @b defines/binding FieldB
+  long b;
+};
+
+//- @U defines/binding UnionU
+union U {
+  //- @x defines/binding FieldX
+  int x;
+  //- @y defines/binding FieldY
+  long y;
+};
 
 template <typename...T>
 void fn(T&&...);
 
 void f() {
   //- @S ref StructS
-  auto s = S{};
+  //- @"1" ref/init FieldA
+  //- !{ _ ref/init FieldB }
+  auto s = S{1};
```

Comment:
Probably want another test to check that multiple fields can be `ref/init`'d

Reply:
Done.  Also include a test of base-class aggregates.

Reply:
Spoke too soon, working on it.